app/views/layouts/_footer.html.erb

```diff
@@ -5,9 +5,9 @@
       <p><%= SiteConfig.community_description %></p>
       <div><%= render partial: "layouts/social_media" %></div>
       <%# The following copy is necessary for compatability with the Forem AGPL licence which requires instances to link back to the source. %>
-      <p class="fs-xs color-base-60"><%= community_qualified_name %> copyright <%= copyright_notice %></p>
-      <p class="fs-xs color-base-60">Built on <a href="https://www.forem.com">Forem</a> — the <a href="https://dev.to/t/opensource">open source</a> software that powers <a href="https://dev.to">DEV</a> and other inclusive communities.</p>
-      <p class="fs-xs color-base-60">Made with love and <a href="https://dev.to/t/rails">Ruby on Rails</a>.</p>
+      <p class="fs-xs color-base-70"><%= community_qualified_name %> copyright <%= copyright_notice %></p>
+      <p class="fs-xs color-base-70">Built on <a href="https://www.forem.com">Forem</a> — the <a href="https://dev.to/t/opensource">open source</a> software that powers <a href="https://dev.to">DEV</a> and other inclusive communities.</p>
+      <p class="fs-xs color-base-70">Made with love and <a href="https://dev.to/t/rails">Ruby on Rails</a>.</p>
```

Comment:
We've updated footer design couple days ago, so you may need to reflect that in your PR as well. So the color is no longer defined here, instead it's here now:

https://github.com/forem/forem/blob/master/app/assets/stylesheets/config/_colors.scss#L102